official/vision/beta/configs/common.py

```diff
@@ -42,6 +43,30 @@ class AutoAugment(hyperparams.Config):
   translate_const: float = 250
 
 
+@dataclasses.dataclass
+class RandomErasing(hyperparams.Config):
```

Comment:
Are all the arguments need to be exposed to users? Do you know which ones are useful and which ones are not? We can hide arguments as default values in the implementation.

Reply:
I suggest having at least `probability` (the probability to erase some part of the image) and `max_count` (number of rectangles to "erase") configurable. On the other hand, I did not experiment a lot with those parameters. Hence, this is simply my best guess.

Reply:
Let's keep them as what they are now. We can revisit this part later if we have a better understanding of the importance of each param.